core/src/test/java/com/linecorp/armeria/server/HttpServiceTest.java

```diff
@@ -53,6 +58,21 @@ protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req) {
                                     HttpStatus.OK, MediaType.PLAIN_TEXT_UTF_8, "Hello, %s!", name);
                         }
                     }.decorate(LoggingService.newDecorator()));
+            sb.service("/trailers", new AbstractHttpService() {
+                @Override
+                protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req) throws Exception {
+                    return HttpResponse.of(HttpHeaders.of(HttpStatus.OK),
+                                           HttpData.ofUtf8("trailer"),
+                                           HttpHeaders.of(HttpHeaderNames.of("foo"), "bar"));
```

Comment:
Could you also add a test case without `HttpData`?

Reply:
Added, thanks!